### ISSUE TITLE
[Bug] Grafana hangs in Browser while dashboard shows last 5 minutes

### ISSUE THREAD

**Issue by ben-st** [grafana/grafana#9612]
tl:dr grafana won't show the `now-5m` time in any dashboard with  version: 4.6.0-beta2 and crashes my browser windows, as you can see in the screenshots.
No Problems occur with any other time setting e.g. `now-15m`

Please include this information:
- What Grafana version are you using?
 4.6.0-beta2
- What datasource are you using?
  prometheus
- What OS are you running grafana on?
  the official docker image grafana/grafana:latest
- What did you do?
  i imported the standard dashboard from my prometheus datasource and created a second sample 
  dashboard. 
  if i try to set the time range in the dashboard to`now-5m` my browser freezes with the dashboard 
  loading forever
  tested with firefox and chrome.
  If i set it to e.g. `now-15m` i don't have any problems or freezes
- What was the expected result?
  not freezing/crashing my browser
- What happened instead?
  my browser freezes and after some time i get an error page.
<img width="1678" alt="screen shot 2017-10-21 at 19 15 58" src="https://user-images.githubusercontent.com/18464684/31854091-654b3f72-b694-11e7-8ff2-b3032f661484.png">
<img width="1679" alt="screen shot 2017-10-21 at 19 16 08" src="https://user-images.githubusercontent.com/18464684/31854092-65650e34-b694-11e7-8a54-7361199627d8.png">
<img width="1065" alt="screen shot 2017-10-21 at 19 16 24" src="https://user-images.githubusercontent.com/18464684/31854093-657e1550-b694-11e7-92cb-9c3a50e181ed.png">

**Comment by arvalaan** [comment]
Having the same issue on a freshly installed Grafana instance (Docker)

Grafana v4.6.0-beta2 (commit: fff4cfd)

**Comment by torkelo** [comment]
This has already been reported twice and fixed :) will release new beta or stable on Monday

**Comment by ben-st** [comment]
thx for your feedback :)
